src/firefly/js/charts/ChartsCntlr.js

```diff
@@ -522,6 +525,25 @@ export function getChartDataElement(chartId, chartDataElementId=FIRST_CDEL_ID) {
     return get(flux.getState(), [CHART_SPACE_PATH, 'data', chartId, 'chartDataElements', chartDataElementId]);
 }
 
+/**
+ * Get error object associated with the given chart data element
+ * @param chartId
+ * @returns {Array<{message:string, reason:object}>} an array of error objects
+ */
+export function getErrors(chartId) {
+    const chartDataElements = get(flux.getState(), [CHART_SPACE_PATH, 'data', chartId, 'chartDataElements']);
+    const errors = [];
+    if (chartDataElements) {
+        Object.keys(chartDataElements).forEach((id) => {
+            const error = chartDataElements[id].error;
+            if (error) {
+                errors.push(error);
+            }
+        });
+    }
+    return errors;
+}
```

Comment:
Object.entries() is a good alternative.
const errors = Object.entries(chartDataElements)
                                    .filter( ([k,v]) -> v.error)
                                    .map( ([k,v]) -> v.error);